lib/streamlit/elements/widgets/file_uploader.py

```diff
@@ -137,7 +144,7 @@ class FileUploaderMixin:
     def file_uploader(
         self,
         label: str,
-        type: Optional[Union[str, Sequence[str]]],
+        type: Union[str, Sequence[str], None],
```

Comment:
Yes, this is something I considered, but at the top of this file we have 

```python
SomeUploadedFiles = Union[
    UploadedFile,
    DeletedFile,
    List[Union[UploadedFile, DeletedFile]],
    None,
]
```

and even with `from __future__ import annotations` 
```python
SomeUploadedFiles = UploadedFile | DeletedFile | List[Union[UploadedFile, DeletedFile]] | None
```
doesn't work, because we have assignments and do not just type annotation

To keep it consistent, I do not import annotations for that files, although I don't think it is a very bad idea to not be inconsistent in this case